apps/web/src/services/helpers.ts

```diff
@@ -0,0 +1,43 @@
+type SimpleParam = string | number | boolean
+type UriParam = SimpleParam | SimpleParam[]
+
+function encodeUriParam(value: UriParam) {
+  if (Array.isArray(value)) return value.map(encodeURIComponent).join(',')
+
+  return encodeURIComponent(value)
+}
+
+export function addParameters(
```

Comment:
Why all of this code? `URLSearchParams` already URL encodes parameters

Reply:
Yep